pkg/build/api/validation/validation.go

```diff
@@ -239,7 +240,8 @@ func validateImageChange(imageChange *buildapi.ImageChangeTrigger) errs.Validati
 	if len(imageChange.Image) == 0 {
 		allErrs = append(allErrs, errs.NewFieldRequired("image"))
 	}
-	if len(imageChange.From.Name) == 0 {
+	var emptyObject kapi.ObjectReference
+	if imageChange.From == emptyObject {
```

Comment:
I agree with you but it's [not](https://github.com/openshift/origin/blob/1f5063f903cd5cbf9154c882195ea0a82041896e/pkg/build/api/types.go#L292) a pointer :/